flink-next-gen-connector-influxdb/src/main/java/org/apache/flink/streaming/connectors/influxdb/sink/writer/InfluxDBWriter.java

```diff
@@ -69,7 +69,7 @@ public void write(final IN in, final Context context) throws IOException {
 
     @Override
     public List<IN> snapshotState() throws IOException {
-        return this.elements;
+        return Collections.emptyList();
```

Comment:
Why?

Reply:
```suggestion
        return this.elements;
```

Reply:
Should stay as it is for state management. I guess, though we should discuss with Arvid, that Flink persists the state here and in case of failure reruns these elements. As we buffer 1000 elements, we need to persist these in case of snapshotting.